lib/SVF-FE/LLVMModule.cpp

```diff
@@ -145,10 +148,64 @@ void LLVMModuleSet::build()
             BasicBlock *exitBB =  const_cast<BasicBlock*>(LLVMUtil::getFunExitBB(func));
```

Comment:
remove const_cast

Reply:
exitBB is not a constant field in svffun. thus, we should cast the const value to non-constnt.